pkg/sfu/downtrackspreader.go

```diff
@@ -83,39 +75,17 @@ func (d *DownTrackSpreader) HasDownTrack(subscriberID livekit.ParticipantID) boo
 
 func (d *DownTrackSpreader) Broadcast(writer func(TrackSender)) {
 	downTracks := d.GetDownTracks()
-	if d.params.Threshold == 0 || (len(downTracks)) < d.params.Threshold {
-		// serial - not enough down tracks for parallelization to outweigh overhead
-		for _, dt := range downTracks {
-			writer(dt)
-		}
-	} else {
-		// parallel - enables much more efficient multi-core utilization
-		start := atomic.NewUint64(0)
-		end := uint64(len(downTracks))
-
-		// 100µs is enough to amortize the overhead and provide sufficient load balancing.
-		// WriteRTP takes about 50µs on average, so we write to 2 down tracks per loop.
-		step := uint64(2)
-
-		var wg sync.WaitGroup
-		wg.Add(d.numProcs)
-		for p := 0; p < d.numProcs; p++ {
-			go func() {
-				defer wg.Done()
-				for {
-					n := start.Add(step)
-					if n >= end+step {
-						return
-					}
-
-					for i := n - step; i < n && i < end; i++ {
-						writer(downTracks[i])
-					}
-				}
-			}()
-		}
-		wg.Wait()
+	threshold := uint64(d.params.Threshold)
+	if threshold == 0 {
+		threshold = 1000000
```

Comment:
Should this be 20?

Reply:
the previous code did disable parallel when Threshold is 0, keep it unchange